corehq/apps/users/models.py

```diff
@@ -1582,15 +1581,13 @@ def get_cases(self, deleted=False, last_submitter=False, wrap=True):
             yield CommCareCase.wrap(doc) if wrap else doc
 
     @property
-    def case_count(self):
-        result = CommCareCase.view('case/by_user',
-            startkey=[self.user_id],
-            endkey=[self.user_id, {}], group_level=0
-        ).one()
-        if result:
-            return result['value']
-        else:
-            return 0
+    def analytics_only_case_count(self):
```

Comment:
@dimagi/team-commcare-hq I was talking with @dannyroberts about a convention for functions that aren't guaranteed to have up-to-the-minute data, and am proposing `analytics_only` as a way to mark them as such. This will make it clear when you call it whether you can count on it be 100% up to date.